h2integrate/simulation/technologies/hydrogen/electrolysis/run_h2_PEM.py

```diff
@@ -198,6 +201,45 @@ def run_h2_PEM(
     if not debug_mode:
         h2_ts, h2_tot = clean_up_final_outputs(h2_tot, h2_ts)
 
+    n_stacks_new = int(sum(np.isnan(h2_tot.loc["Stack Life [hours]"].to_list())))
+
+    # if some stacks were never turned on (are "new") there may be inf and nans in some outputs
+    if n_stacks_new > 0:
+        never_on_cluster_list = np.isnan(h2_tot.loc["Stack Life [hours]"].to_list())
+        user_defined_pem_param_dictionary.setdefault("curve_coeff", None)
+        pem = create_1MW_reference_PEM(curve_coeff=user_defined_pem_param_dictionary["curve_coeff"])
+        # Update: 'Time Until Replacement [hrs]', 'Life: Efficiency [kWh/kg]', and
+        # 'Life: Efficiency [%-HHV]'
+        # In Performance Schedules, update "Annual Average Efficiency [kWh/kg]" and
+        # 'Annual Average Efficiency [%-HHV]'
+
+        annual_eff_kWh_pr_kg = np.zeros((n_pem_clusters - n_stacks_new, int(useful_life)))
+        ci = 0
+        for never_on, cluster in zip(
+            never_on_cluster_list, h2_tot.loc["Performance By Year"].index.to_list()
+        ):
+            if not never_on:
```

Comment:
Non-blocking nitpick: Could you refactor this to avoid the semantics of `not never_on`? The [double negative](https://www.showwcase.com/article/33668/code-smell-51-double-negatives) might make it confusing for future devs at first glance.